docs/proposals/20191017-kubeadm-based-control-plane.md

```diff
@@ -346,58 +376,72 @@ spec:
 - Completely removing the control plane and issuing a delete on the underlying machines.
 - User documentation should focus on deletion of the Cluster resource rather than the KubeadmControlPlane resource.
 
-##### KubeadmControlPlane rollout (using create-swap-and-delete)
+##### KubeadmControlPlane rollout
 
-- Triggered by: 
-    - Changes to Version
-    - Changes to the kubeadmConfigSpec
-    - Changes to the infrastructureRef
-    - The `upgradeAfter` field, which can be set to a specific time in the future
-        - Set to `nil` or the zero value of `time.Time` if no upgrades are desired
-        - An upgrade will run when that timestamp is passed
-        - Good for scheduling upgrades/SLOs
-        - Set `upgradeAfter` to now (in RFC3339 form) if an upgrade is required immediately
-
-- Rollout operations rely on scale up and scale down which are be blocked based on Etcd and control plane health checks
+KubeadmControlPlane rollout operations rely on [scale up](#scale up) and [scale down](#scale_down) which are be blocked based on Etcd and control plane health checks
   - See [Health checks](#Health checks) below.
 
+KubeadmControlPlane rollout is triggered by:
+  - Changes to Version
+  - Changes to the kubeadmConfigSpec
+  - Changes to the infrastructureRef
+
+##### KubeadmControlPlane rollout (using create-swap-and-delete)
+
```

Comment:
What about adding a sentence that states that this is paragraph describes how the rollout process is performed if UpgradeStrategy is empty or scale out.
A similar sentence should be added at the beginning of the next paragraph

Reply:
Do we want to set UpgradeStrategy to scale out? I was thinking that when UpgradeStrategy is not set KCP uses the default rollout strategy and scale out. But if user wants to set UpgradeStrategy, scale in would be the only possible value.